media/livestream/createChannel.js

```diff
@@ -1,10 +1,11 @@
 /**
- * Copyright 2022, Google, Inc.
+ * Copyright 2024 Google LLC
```

Comment:
You shouldn't update the year here as you're updating a sample

Reply:
Reverted